mysite/settings.py

```diff
@@ -10,19 +10,22 @@
 https://docs.djangoproject.com/en/2.0/ref/settings/
 """
 
-import os
+import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = environ.Path(__file__) - 2  # 2 folder back
+ENV = environ.Env(DEBUG=(bool, False), )  # set default values and casting
```

Comment:
It's not a constant.

Reply:
@webknjaz ```hookid: pylint
************* Module mysite.settings
mysite/settings.py:L  17:(invalid-name) Constant name "env" doesn't conform to UPPER_CASE naming style (C  0), ::C0103```
I need just add `env` to `good-names` in `.pre-commit-config.yaml`, right?

Reply:
@webknjaz Done.